docs/source/virtual_ecosystem/theory/soil/environmental_links.md

```diff
@@ -25,22 +25,70 @@ language_info:
 
 # Environmental impacts on soil processes
 
-Litter decay and soil nutrient transformations are both affected by temperature. As the
+Litter decay and soil nutrient transformations are both affected by environment. As the
 soil model explicitly includes microbes, temperature effects many different processes in
 the model, e.g. enzymatic rates and the carbon use efficiency of microbial growth.
 Temperature is more straightforward in the litter model and just effects the decay rate
 of each litter pool. Processes that take place underground are also affected by soil
 moisture. For the soil moisture response, an empirical relationship is used for both
-litter decay and soil organic matter breakdown.
+litter decay and soil organic matter breakdown. For the enzymes in the soil model, their
+action is also affected by soil pH and clay fraction. Finally, the rate at which
+nutrients leach from the soil is affected by the rate at which water flows downwards
+through the soil. We will now describe each of these in more detail.
 
-:::{admonition} In progress 🛠️
+## Microbial response to temperature
 
-The representation of soil microbial communities has still not been finalised. Once this
-has happened this section of the documentation will be extended to detail the relevant
-thermal responses.
+Temperature is one of the most significant drivers of microbial processes. In the soil
+model, two different approaches are taken to model the effects of temperature on
+microbial process rates. While most processes are modelled using the Arrhenius equation,
+a different approach is taken to modelling the efficiency of microbial growth.
+
+### Arrhenius equation
+
+:::{admonition} Future directions 🔭
+
+The Arrhenius equation is a simple model for the impact of temperature on biological
+rates. We are using this equation as a simple initial approach to incorporate
+temperature in the model, and anticipate deprecating it in favour of more refined models
+in future.
 
 :::
 
+We model the thermal variation of the following processes using the Arrhenius equation:
+
+* microbial biomass loss
+* microbial uptake rate
+* microbial uptake saturation
+* enzyme rate
+* enzyme saturation
+
+The form of the equation is as follows
+
+$$f(T) = \exp{\frac{-E_a}{R} * (\frac{1}{T} - \frac{1}{T_{\mathrm{ref}}})},$$
+
+where $E_a$ the activation energy of the process of interest, $R$ is the molar gas
+constant, $T$ is the environmental temperature, and $T_{\mathrm{ref}}$ the reference
+temperature.
```

Comment:
Note for future, we should add units throughout the documentations, to remind ourselves and for the readers.